DrDocket/Spider.cpp

```diff
@@ -1,4 +1,5 @@
 #include "Spider.h"
+#include "Appointment.h"
```

Comment:
Spider header includes the Resource header which includes the Appointment header. A comment on this dependency could have been left by me though.